dev/PowerNotifications/PowerNotifications.cpp

```diff
@@ -197,32 +198,64 @@ namespace winrt::Microsoft::ProjectReunion::implementation
         check_hresult(UnregisterSystemIdleStatusChangedListener(
             make_self<factory_implementation::PowerManager>()->m_systemIdleStatusHandle));
     }
-   
-    // PowerSchemePersonality Functions
-    EventType& PowerSchemePersonality_Event()
+
+    bool IsEffectivePowerModeV2Supported()
```

Comment:
I went ahead with that because checking if V2 is supported is what we were doing. But I guess since we don't know if new versions are going to be added later, we can incorporate this. Changed.